src/App.tsx

```diff
@@ -5,13 +5,14 @@ import Home from "./components/home/Home";
 import Dashboard from "./components/dashboard/Dashboard";
 import Invoices from "./components/invoices/Invoices";
 import { PATHS } from "./urls/urls";
-import { Login } from "./components/login/Login";
+import { Register } from "./components/register/Register";
 import { useState } from "react";
+import { Login } from "./components/login/Login";
 
 function App() {
   //todo // state jest tylko po to aby przetestować ekran login
   const [state, setState] = useState(true);
-  const { home, dashboard, invoices, login } = PATHS;
+  const { home, dashboard, invoices, login, register } = PATHS;
   return (
     <div className="app">
       {state ? (
```

Comment:
Jeżeli poprzedni komentarz jest prawdą, to ja bym tutaj podszedł do tego inaczej. 
Zwykle w komercyjnych projektach mamy takie podejście że:

Tworzymy dodatkowy komponent np. ProtectedRoute który jako parametr przyjmuje komponent strony np dashboard, invoices etc. Wewnatrz ProtectedRoute, sprawdzany jest stan zalogowania oraz np. permissiony (mozemy je tez przekazywać do komponentu). Jezeli wszystko sie zgadza, wtedy ProtectedRoute zwraca ten komponent, a jesli nie to robi redirect na login page.

Np.: `<ProtectedRoute component={<Dashboard />} requiredRoles={[Roles.ADMIN]} />`

Może coś takiego byłoby tutaj spoko zrobić? Mielibysmy zabezpieczone strony wymagajace zalogowania oraz odpowiednich permissionow.